### PR TITLE
[Text Analytics] Add pagination by using x-ms-paths

### PR DESCRIPTION
**Pull request by maririos**
Adding for convenience on the SDKS so we can get codegen to generate the pagination for us.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json [hunk 1]
```diff
@@ -149,6 +149,9 @@
             "$ref": ".//examples//SuccessfulAnalyzeStatusRequest.json"
           }
         },
+        "x-ms-pageable": {
+          "operationName": "AnalyzeStatusNextPage"
```
Comment (mssfang) [inline specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json:153]: missing nextLinkName and itemName

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json [hunk 2]
```diff
@@ -670,6 +676,110 @@
       }
     }
   },
+  "x-ms-paths": {
+    "/entities/health/jobs/{nextLink}": {
+      "get": {
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "values"
```
Comment (mssfang) [inline specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json:684]: add operationName as well

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json (hunk 1 above)
Comment (maririos) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json:153]: thanks Shawn. Interesting... .NET doesn't need it. At the same time, adding it didn't affect .NET so I will update the PR

Comment (maririos) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json]: Added. Could you try again ?

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json [hunk 3]
```diff
@@ -149,6 +149,11 @@
             "$ref": ".//examples//SuccessfulAnalyzeStatusRequest.json"
           }
         },
+        "x-ms-pageable": {
+          "operationName": "AnalyzeStatusNextPage",
+          "nextLinkName": "nextLink",
```
Comment (iscai-msft) [inline specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json:154]: For some of these next links, the name should actually be `@nextLink` based off of swagger definitions

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json [hunk 4]
```diff
@@ -149,6 +149,11 @@
...
+          "itemName": "values"
```
Comment (iscai-msft) [inline specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json:155]: There is no property on the response object called `values`, I think it should be `tasks` in this case

File: specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json (hunk 3 above)
Comment (ShivangiReja) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json:154]: @maririos I think what @iscai-msft is trying to say is "nextLink" should be a property name defined [here](https://github.com/Azure/azure-rest-api-specs/blob/cbaf1c13997c8159cc10615fea27b32c20db676c/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L2328), which in this case is `"@nextLink"`, so we should update `nextLink` to `@nextLink`.

Comment (ShivangiReja) [reply specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json:154]: @maririos I think what @iscai-msft is trying to say is "nextLink" should be a property name defined in response object [here](https://github.com/Azure/azure-rest-api-specs/blob/cbaf1c13997c8159cc10615fea27b32c20db676c/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L2328), which in this case is `"@nextLink"`, so we should update `nextLink` to `@nextLink`.

**Comment by maririos** [comment]
because of how data is returned it is not possible for other languages to get pagination correctly as there is no `values` or equivalent property for `itemName`. Closing

**Pull request closed without merging**